mars/web/server.py

```diff
@@ -141,7 +142,23 @@ def write_mutable_tensor(self, session_id, name, body):
         index_json_size = np.asscalar(np.frombuffer(body[0:8], dtype=np.int64))
         index_json = json.loads(body[8:8+index_json_size].decode('ascii'))
         index = Indexes.from_json(index_json).indexes
-        value = dataserializer.loads(body[8+index_json_size:], raw=False)
+        if payload_type is None:
+            value = dataserializer.loads(body[8+index_json_size:], raw=False)
+        elif payload_type == 'tensor':
+            tensor_chunk_offset = 8 + index_json_size
+            with pyarrow.BufferReader(body[tensor_chunk_offset:]) as reader:
+                value = pyarrow.read_tensor(reader).to_numpy()
+        elif payload_type == 'record_batch':
+            schema_size = np.asscalar(np.frombuffer(body[8+index_json_size:8+index_json_size+8], dtype=np.int64))
+            schema_offset = 8 + index_json_size + 8
+            with pyarrow.BufferReader(body[schema_offset:schema_offset+schema_size]) as reader:
+                schema = pyarrow.read_schema(reader)
```

Comment:
Does schema required each time we write a record batch?

Reply:
It is more convenient to include schema in every batch, otherwise, we may need to record it somewhere. Compared to the size of payload data, the size of schema is ignorable, IMHO.